app/transactions/ams/steps/create_aapb_admin_data.rb

```diff
@@ -74,7 +74,7 @@ def set_annotations_attributes(admin_data, change_set)
         return if change_set.fields["annotations"].nil?
         change_set.fields["annotations"].each do |annotation|
           next if annotation["_destroy"] == "true"
-          next if annotation.except("id").values.all?(&:empty?)
+          next if annotation.except("id").values.all? { |val| val.nil? || val.empty? }
```

Comment:
non-blocking but i think instead of 
```rb
next if annotation.except("id").values.all? { |val| val.nil? || val.empty? }
``` 
you can shorten it to 
```rb
next if annotation.except("id").values.all?(&:blank?)
```

Reply:
Yes of course! I used blank in my other PR, don't know why I didn't use it hear